test/system/800-scenario.bats

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bats 
+#
+# Test for the scenario that
+# "podman import" tarball exported from a container and modified the content.
+# 
+
+load helpers
+
+alpine="quay.io/libpod/alpine"
+
+@test "Implied pull, build, export, modify, import, tag, run, kill" {
+
+  # Create a test file following test
+  touch $PODMAN_TMPDIR/testfile1
+  echo "modified tar file" >> $PODMAN_TMPDIR/testfile2
+  
+  # Create Dockerfile for test
+  dockerfile=$PODMAN_TMPDIR/Dockerfile
+  
+  cat >$dockerfile <<EOF
+FROM $alpine
+ADD testfile1 /tmp
+WORKDIR /tmp
+EOF
+
+  b_img=before_change_img
+  b_cnt=before_change_cnt
+  a_img=after_change_img
+  a_cnt=after_change_cnt
+
+  # Build from Dockerfile FROM non-existing local image
+  run_podman build -t $b_img $PODMAN_TMPDIR
+  run_podman run -d --name $b_cnt --stop-signal SIGKILL $b_img sleep 60
+
+  # Export built container as tarball
+  run_podman export -o $PODMAN_TMPDIR/$b_cnt.tar $b_cnt
+  run_podman rm -fa
+
+  # Modify tarball contents
+  tar --delete -f $PODMAN_TMPDIR/$b_cnt.tar tmp/testfile1
+  tar -rf $PODMAN_TMPDIR/$b_cnt.tar $PODMAN_TMPDIR/testfile2
+
+  # Import tarball
+  run_podman import -q $PODMAN_TMPDIR/$b_cnt.tar \
+      --change "CMD sh -c \
+      \"trap 'exit 33' 2;
+      echo READY;
+      cat $PODMAN_TMPDIR/testfile1 2>&1;
+      cat $PODMAN_TMPDIR/testfile2;
+      while true; do sleep 0.05;done\""
+  iid=$output
+
+  # Tag imported image
+  run_podman tag $iid $a_img
```

Comment:
Note: `podman import $PODMAN_TMPDIR/$b_cnt.tar ... $a_img` would tag the imported image.